app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardDatastoresStep/MappingWizardDatastoresStep.js

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { noop, Button, bindMethods } from 'patternfly-react';
+
+// import DualPaneMapper from '../DualPaneMapper/DualPaneMapper';
+// import DualPaneMapperList from '../DualPaneMapper/DualPaneMapperList';
+// import DualPaneMapperCount from '../DualPaneMapper/DualPaneMapperCount';
+// import DualPaneMapperListItem from '../DualPaneMapper/DualPaneMapperListItem';
+
+class MappingWizardDatastoresStep extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      selectedCluster: null, // dropdown selected cluster
+      selectedTargetDatastore: null, // eslint-disable-line react/no-unused-state
+      selectedSourceDatastore: [], // eslint-disable-line react/no-unused-state
+      datastoreMappings: [], // eslint-disable-line react/no-unused-state
+      selectedDatastoreMapping: null // eslint-disable-line react/no-unused-state
+    };
+
+    bindMethods(this, ['selectSourceCluster']);
+  }
+
+  componentDidMount() {}
+
+  selectSourceCluster(id) {
+    // when dropdown selection occurs for source cluster, we go retrieve the datastores for that
+    // cluster
+    const { fetchDatastoresUrl, fetchDatastoresAction } = this.props;
+    fetchDatastoresAction(fetchDatastoresUrl, id);
```

Comment:
this is going to become `(url, sourceClusterId, [...targetClusterIds])` ... and ill gather all of them for you...fixing it now..